docs/cody/clients/install-vscode.mdx

```diff
@@ -311,9 +311,28 @@ Cody also works with Cursor, Gitpod, IDX, and other similar VS Code forks. To ac
 
 ## Supported LLM models
 
-Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands. Users on Cody **Pro** can choose from a list of supported LLM models for Chat, Commands, and Autocomplete. These LLMs are Claude Instant, Claude 2.0, Claude 2.1, Claude 3 (Haiku, Opus and Sonnet), Claude 3.5 Sonnet, ChatGPT 3.5 Turbo, ChatGPT 4 Turbo Preview, GPT-4o, Mixtral, Google Gemini 1.5 Pro and Google Gemini 1.5 Flash.
+Claude Sonnet 3 is the default LLM model for Cody Free users for Chat and Commands. Users on Cody **Free** and **Pro** can choose from a list of supported LLM models for Chat and Commands. Here's a preview:
```

Comment:
Should we just add a link to the token-limit.mdx page instead so that we don't have to update two separate pages everytime we add a new model? Something like 
Check out the latest LLM models available in Cody in the Token Limit page

Reply:
Yes, totally; I'll add a link.